grails-app/conf/DBMS.properties

```diff
@@ -2,7 +2,7 @@ INFORMIX                 = 1
 DB                       = 1
 DW_DATASOURCE_NAME       = java:DW
 OLTP_DATASOURCE_NAME     = java:comp/env/OLTP
-JTS_OLTP_DATASOURCE_NAME = java:JTS_OLTP
+JTS_OLTP_DATASOURCE_NAME = java:comp/env/OLTP
```

Comment:
I don't know the background some dsn point to same datasource. I choose change the config file instead of the code. But it's up to you if you want otherwise